server/rpc_server.go

```diff
@@ -131,8 +131,6 @@ func (s *rpcServer) HandleEvent(e broker.Event) error {
 
 // ServeConn serves a single connection
 func (s *rpcServer) ServeConn(sock transport.Socket) {
```

Comment:
This is a really long func，may be you can refactor it

Reply:
It's not worth the time right now